src/components/WebexAvatar/README.md

```diff
@@ -1,9 +1,22 @@
 # Webex Avatar Component
+Webex avatar component displays the avatar and status of a _Webex Teams_ user.
 
-Webex avatar component displays the avatar of a _Webex Teams_ user with a given `personID` and `adapter` object.
+![Webex Avatar](./WebexAvatar.png)
 
-## How to use
-
-```js
-<WebexAvatar personID="PersonID" adapter={peopleJSONAdapter} />
+## Preview
+To see all the different possible states of the Webex Avatar component, you can run our storybook
+```shell
+  npm start
 ```
+
+## Embed
+1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
+    ```js
+      new jsonAdapter = new PeopleJSONAdapter()
+    ```
+2. Create a component instance by passing the person ID and the component data adapter
+    ```js
+      <WebexAvatar personID="personID" adapter={jsonAdapter} />
+    ```
+
+The component knows how to manage its data. If anything changes in the data source that the adapter manages, the component will also update on its own.
```

Comment:
Maybe another link to the PeopleAdapter?

Add newline at the end please

Reply:
This was meant to be a general statement to be used throughout all the component READMEs. Like a little reminder